src/Qowaiv/Extensions/System.DateOnly.cs

```diff
@@ -0,0 +1,97 @@
+﻿#if NET6_0_OR_GREATER
+
+using Qowaiv;
+
+namespace System;
+
+/// <summary>Extensions on <see cref="DateOnly"/>.</summary>
+public static class QowaivDateOnlyExtensions
+{
+    /// <summary>Returns a new date that adds the value of the specified <see cref="DateSpan"/>
+    /// to the value of this instance.
+    /// </summary>
+    /// <param name="d">
+    /// The date time to add a <see cref="DateSpan"/> to.
+    /// </param>
+    /// <param name="value">
+    /// A <see cref="DateSpan"/> object that represents a positive or negative time interval.
+    /// </param>
+    /// <returns>
+    /// A new date whose value is the sum of the date represented
+    /// by this instance and the time interval represented by value.
+    /// </returns>
+    /// <exception cref="ArgumentOutOfRangeException">
+    /// The resulting date is less than <see cref="DateOnly.MinValue"/> or greater
+    /// than <see cref="DateOnly.MaxValue"/>.
+    /// </exception>
+    [Pure]
+    public static DateOnly Add(this DateOnly d, DateSpan value) => d.Add(value, false);
+
+    /// <summary>Returns a new local date time that adds the value of the specified <see cref="DateSpan"/>
+    /// to the value of this instance.
+    /// </summary>
+    /// <param name="d">
+    /// The date to add a <see cref="DateSpan"/> to.
+    /// </param>
+    /// <param name="value">
+    /// A <see cref="DateSpan"/> object that represents a positive or negative time interval.
+    /// </param>
+    /// <param name="daysFirst">
+    /// If true, days are added first, otherwise months are added first.
+    /// </param>
+    /// <returns>
+    /// A new date whose value is the sum of the date and time represented
+    /// by this instance and the time interval represented by value.
+    /// </returns>
+    /// <exception cref="ArgumentOutOfRangeException">
+    /// The resulting date is less than <see cref="DateOnly.MinValue"/> or greater
+    /// than <see cref="DateOnly.MaxValue"/>.
+    /// </exception>
+    [Pure]
+    public static DateOnly Add(this DateOnly d, DateSpan value, bool daysFirst)
```

Comment:
I see how this method solves a problem. But I don't think it makes clear what happens to the reader of calling code.

Reply:
This is the same API as for Date (and DateTime). So changing is a breaking change.

Reply:
This has been addressed with #290 .